### DISCUSSION TITLE
[Feature Request] Do not allow to enter arbitrary values in multi-select template variables

### DISCUSSION THREAD

**Discussion by swachter** [grafana/grafana#42333] [Dashboards, Panels & Visualizations]
ATM a user can enter an arbitrary value in the input field of a multi-select template variable. In case that only predefined selection options are valid, only selection of values should be allowed and direct entering of values should be disabled.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
